cds-snc/cppd-medical-report
# Style medications screen and Delete medications

# Description

NOTE: this is based off #69 (dude!), merge that one first

Style the `medication` list screen and enable deletes

# Test Instructions

- Create some conditions and some medications
- Observe the new medications screen
- Try deleting one!
